camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlRecordReader.java

```diff
@@ -116,11 +126,18 @@ public synchronized void close() throws IOException {
     }
   }
 
-  private CamusWrapper getWrappedRecord(String topicName, byte[] payload) throws IOException {
+  private CamusWrapper getWrappedRecord(String topicName, byte[] payload)
+      throws IOException {
     CamusWrapper r = null;
     try {
       r = decoder.decode(payload);
+    } catch (SchemaNotFoundException e) {
+      mapperContext.getCounter(MSG_SKIPPED.SCHEMA_NOT_FOUND).increment(1);
```

Comment:
How do we use these counter values later?  I think one of the requirement is to fail the job when the skip percentage is above certain threshold.